src/diagram-api/diagram-orchestration.ts

```diff
@@ -19,14 +20,10 @@ import gitGraphDetector from '../diagrams/git/gitGraphDetector';
 //   mindmapRenderer,
 //   mindmapDetector
 // );
-const addDiagrams = () => {
+export const addDiagrams = () => {
   registerDiagram(
     'gitGraph',
-    gitGraphParser,
-    gitGraphDb,
-    gitGraphRenderer,
-    undefined,
+    { parser: gitGraphParser, db: gitGraphDb, renderer: gitGraphRenderer },
     gitGraphDetector
   );
 };
-export default addDiagrams;
```

Comment:
Decoupling will continue here

Reply:
Only removed the default export. Named export is still there.